<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MRTwindow</class>
 <widget class="QMainWindow" name="MRTwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MRTwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="control" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>1st motor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Axis" name="axis1" native="true"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="use2nd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>2nd motor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Axis" name="axis2" native="true"/>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>On completion move the motor(s) to:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="after">
         <property name="toolTip">
          <string>Where the positioners are moved after the scan is complete</string>
         </property>
         <item>
          <property name="text">
           <string>End position</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Start position</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prior position</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Fast shutter control:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="placeShutter"/>
       </item>
       <item>
        <widget class="QLabel" name="disabledShutter">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Exposure cannot be started while shutter is opened.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="badShutter">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Inconsistent shutter parameters.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar"/>
    </item>
    <item>
     <widget class="QPushButton" name="start">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Axis</class>
   <extends>QWidget</extends>
   <header>axis.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>use2nd</sender>
   <signal>toggled(bool)</signal>
   <receiver>axis2</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
